ksql-engine/src/main/java/io/confluent/ksql/structured/SchemaKStream.java

```diff
@@ -174,7 +176,9 @@ public SchemaKStream leftJoin(final SchemaKTable schemaKTable,
               }
 
               return new GenericRow(columns);
-            }, Joined.with(Serdes.String(), SerDeUtil.getRowSerDe(joinSerDe, this.getSchema()), null));
+            }, Joined.with(Serdes.String(), SerDeUtil.getRowSerDe(joinSerDe, this.getSchema(),
+                                                                  ksqlConfig, false),
+                           null));
```

Comment:
nit: bad spacing, `null` needn't be on a new line. It's very easy to miss while reading.

Reply:
Yes, intelliJ made it this way. Fixed it.